blazor/rich-text-editor/tools/dialogSettings.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: Customization of Blazor Rich Text Editor Dialog | Syncfusion
+description: Check out and learn here all about DialogSettings in Syncfusion Blazor RichTextEditor component and more.
+platform: Blazor
+control: RichTextEditor
+documentation: ug
+---
+
+# Customization of Blazor Rich Text Editor Dialog
+
+The integration of the child component allows for improved customization of the RichTextEditor in the Blazor component. With this, you can now directly control the RichTextEditor's dialogs.
```

Comment:
Content is not clear, can you reframe it?

Reply:
Addressed